Makefile

```diff
@@ -27,8 +32,32 @@ format-style:
 	@# `make` needs `$$` to output `$`. Ref: http://stackoverflow.com/questions/2382764.
 	autopep8 `git ls-files | grep "\.py$$"`
 
-test: clean check-syntax-errors check-style check-types
-	env PYTEST_ADDOPTS="$$PYTEST_ADDOPTS --cov=openfisca_core" pytest
+## Run openfisca-core & country/extension template tests.
+test:: clean check-syntax-errors check-style check-types
+test:: \
+	$(shell git ls-files "tests/*.py") \
+	${python_packages}/${country_template}/tests \
+	${python_packages}/${extension_template}/tests
```

Comment:
After running the tests locally, I believe it would add a lot to readability if the Core tests were run separately from the Country and Extension templates, both here and in CI.
Could we create dedicated targets for each of those, with `test` running all three? 🙂